pkg/kvrpcpb/kvrpcpb.pb.go

```diff
@@ -53,7 +53,7 @@ func (x CommandPri) String() string {
 	return proto.EnumName(CommandPri_name, int32(x))
 }
 func (CommandPri) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_kvrpcpb_da15c4d6242f2aec, []int{0}
+	return fileDescriptor_kvrpcpb_e53caf779f8a2335, []int{0}
```

Comment:
Looks not related changes. Better to reset these files.

Reply:
In fact I change a comment in `proto/kvrpcpb.proto`. I think it's caused by that. So how about keeping it because other guys can also meet this change after he runs `make go`.

Reply:
ok. It seems that the sha code are random generated...